acpreprocessing/stitching_modules/convert_to_n5/tiff_to_ngff.py

```diff
@@ -598,16 +632,243 @@ def write_mimgfns_to_n5(
                     mimgfns, max_mip, slice_length, mip_dsfactor,
                     lvl_to_mip_kwargs=lvl_to_mip_kwargs,
                     interleaved_channels=interleaved_channels,
-                    channel=channel):
+                    channel=channel,deskew_kwargs=deskew_kwargs):
                 futs.append(e.submit(
                     dswrite_chunk, mip_ds[miparr.lvl],
                     miparr.start, miparr.end, miparr.array))
             for fut in concurrent.futures.as_completed(futs):
                 _ = fut.result()
 
+def omezarr_attrs(name,position_xyz,lvl0_xyz_res,max_lvl):
```

Comment:
Definitely not blocking, but this feels like it should be a schematized object that we build somewhere else and serialize (using external library?)